sdk/attestation/Azure.Security.Attestation/src/AttestationToken.cs

```diff
@@ -25,7 +29,7 @@ public class AttestationToken
         /// Initializes a new instance of the <see cref="AttestationToken"/> class.
         /// </summary>
         /// <param name="token">string JWT to initialize.</param>
-        internal AttestationToken(string token)
+        internal protected AttestationToken(string token)
```

Comment:
I'm not sure if the intention for `protected` was meant to enable customer scenarios for this constructor, but I wanted to mention it being promoted to the public API surface.   
_(I often forget this and it recently came back to bite me in one case; no offense intended if it was already on your radar.)_

If you weren't intending it for customer scenarios, we may want to consider tagging it as non-visible to the editor (`[EditorBrowsable(EditorBrowsableState.Never)]`) so that it isn't as easily discoverable through code-completion and MS Docs, and adding `<remarks>`  that clarify its intent, to warn away the adventurous.

Reply:
For test (mocking) scenarios, it can be incredibly useful to instantiate an AttestationToken from a string (the JSON Web Token returned by the service). But it's not a mainstream scenario for the service. Let me add EditorBrowsable and add a remarks to explain the use of this pattern.